### PR TITLE
[Refactor] Remove unwraps from tauri

### PR DESCRIPTION
**Pull request by tensor-programming**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] New Binding Issue #___
- [x] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

Remove unwrap calls which are not helpful and expressive.  Allows the code to be a bit more self documenting and allows end user to identify where an error has occurred.

### CODE REVIEW

File: cli/tauri-cli/src/bundle/common.rs [hunk 1]
```diff
@@ -268,42 +269,44 @@ mod tests {
...
       std::fs::read(tmp.path().join("parent/copy/sub/file.txt"))
-        .unwrap()
+        .expect("Failed to read file")
         .as_slice(),
       b"Hello, world!\n"
     );
     assert!(tmp.path().join("parent/copy/link").exists());
     assert_eq!(
-      std::fs::read_link(tmp.path().join("parent/copy/link")).unwrap(),
+      std::fs::read_link(tmp.path().join("parent/copy/link")).expect("Failed to read from simlink"),
```
Comment (nothingismagick) [inline cli/tauri-cli/src/bundle/common.rs]: symlink i believe

File: cli/tauri-cli/src/bundle/osx_bundle.rs [hunk 2]
```diff
@@ -201,7 +201,7 @@ fn copy_frameworks_to_bundle(bundle_directory: &Path, settings: &Settings) -> cr
   for framework in frameworks.iter() {
     if framework.ends_with(".framework") {
       let src_path = PathBuf::from(framework);
-      let src_name = src_path.file_name().unwrap();
+      let src_name = src_path.file_name().expect("Couldn't get file name");
```
Comment (nothingismagick) [inline cli/tauri-cli/src/bundle/osx_bundle.rs]:
```
Could not get filename
```
Like in L248 in this file

File: cli/tauri-cli/src/bundle/common.rs (hunk 1 above)
Comment (tensor-programming) [reply cli/tauri-cli/src/bundle/common.rs]: Alright, fixed.

File: cli/tauri-cli/src/bundle/osx_bundle.rs (hunk 2 above)
Comment (tensor-programming) [reply cli/tauri-cli/src/bundle/osx_bundle.rs]: alright, differentiated them with icon and framework respectively.

**Pull request merged**